### ISSUE TITLE
 TaskJointPosVelAccBounds: Swapped default values for qMin/qMax

### ISSUE THREAD

**Issue by paulinejmaurice** [stack-of-tasks/tsid#158]
In the constructor of  the TaskJointPosVelAccBounds class, qMin is set to +1e10 (positive value) and qMax -1e10 (negative value), such that qMin > qMax. I believe this should be the opposite, right?
This is not crucial as long as position limits are reset after (which is the normal use), but in case someone were to set only velocity limits and use default position limits, this might lead to an infeasible problem.

**Comment by andreadelprete** [comment]
You are right @paulinejmaurice ! Thanks for spotting this. Can you do a PR to fix this?
